acceptance/restorer_test.go

```diff
@@ -14,9 +14,9 @@ import (
 	"github.com/sclevine/spec"
 	"github.com/sclevine/spec/report"
 
-	"github.com/buildpacks/lifecycle"
 	"github.com/buildpacks/lifecycle/api"
 	"github.com/buildpacks/lifecycle/cmd"
+	"github.com/buildpacks/lifecycle/lifecycle"
```

Comment:
What do you think about moving things to a new package called `internal`? I dislike the `lifecycle/lifecycle` in the imports. Are the things under `lifecycle` meant for to be public APIs? I don't actually know our stance. I know we use types from various places like `github.com/buildpacks/lifecycle/platform`, which reads nicely.

```suggestion
	"github.com/buildpacks/lifecycle/internal"
```

Reply:
I believe `lifecycle.Rebaser` is imported by pack and kpack ...not sure about the other phases though

Reply:
What about `github.com/buildpacks/lifecycle/phase`? Or making all but the rebaser internal?

Reply:
Maybe it could be `github.com/buildpacks/lifecycle/internal/build` and `github.com/buildpacks/lifecycle/rebase` ...or something

Reply:
I like `github.com/buildpacks/lifecycle/rebase` or `github.com/buildpacks/lifecycle/phase` fine!

In case we want to visualize what this looks like before deciding...

```golang
rebaser := rebase.Rebaser{}
rebaser := rebase.New(opts)

rebaser := phase.Rebaser{}
rebaser := phase.NewRebaser(opts)

// alternatives
rebaser := platform.NewRebaser(opts)
rebaser := core.NewRebaser(opts)
rebaser := buildpacksio.NewRebaser(opts)
rebaser := exec.NewRebaser(opts)
rebaser := stage.NewRebaser(opts)
```

The whole `internal` folder thing golang does can be a bit aggressive. If we want to make sure we don't accidentally block out users who are importing things we don't suspect, I think we should probably avoid nesting under `internal`.